httpcore5/src/main/java/org/apache/hc/core5/net/PercentCodec.java

```diff
@@ -84,6 +84,16 @@ public class PercentCodec {
         URIC.or(UNRESERVED);
     }
 
+    static final BitSet FRAGMENT_SAFE = new BitSet(256);
```

Comment:
@arturobernalg `FRAGMENT_SAFE` does not seem to be used anymore. As well as  `URIC`. They should probably be removed.

Reply:
Removed